dashboard/app/views/scripts/show.html.haml

```diff
@@ -1,7 +1,7 @@
 -# This variable will get any data we need to pass along to scriptOverview.js
 - scriptOverviewData = { scriptData: @script.summarize }
 
-- if current_user && current_user.show_race_interstitial?
```

Comment:
This was the real bug here, I missed updating this when I updated the index.html.haml version